tests/easier68k/core/opcodes/test_move.py

```diff
@@ -0,0 +1,39 @@
+"""
+Test methods for the move command
+"""
+
+from easier68k.simulator.m68k import M68K
+from easier68k.core.opcodes.move import Move
+from easier68k.core.enum.ea_mode import EAMode
+from easier68k.core.models.assembly_parameter import AssemblyParameter
+from easier68k.core.enum.register import Register
+
+def test_move():
+    """
+    Test to see that move works as intended
+    :return:
+    """
+
+    # make a simulator class
+    a = M68K()
+
+    # test immediate -> data register
+
+    # move 123 to D2
+    src = AssemblyParameter(EAMode.IMM, 123)
+    dst = AssemblyParameter(EAMode.DRD, 2)
+
+    # make a testing move command
+    mv = Move(src, dst, 'B')
+
+    mv.execute(a)
+
+    assert a.get_register_value(Register.D2) == 123
+
+def test_move_invalid_behavior():
```

Comment:
It appears you have an incomplete test?

Reply:
Me and @Chris-Johnston have agreed that this will be sorted out later for the sake of merging this branch ASAP

Reply:
Do It!

Reply:
Yup, I'm implementing the ERR change for him right now (he's busy on another project) and then I'm going for it